README.md

```diff
@@ -162,7 +162,7 @@ You can pass any custom pikaday option through the component like this
 
 ```handlebars
 <label>
-  {{pikaday-input options=(hash numberOfMonths=2 disableWeekends=true disableDayFn=(action 'someAction'))}}
+  <PikadayInput @options={{hash numberOfMonths=2 disableWeekends=true disableDayFn={{action 'someAction'}}}}/>
```

Comment:
I think we still need to use `(action 'someAction')` because it is a nested handlebars expression.

Reply:
Thanks for catching this 👍 update made! :)